plugins/foreman_statistics/1.0/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: plugin
+title: Foreman Statistics 1.0 Manual
+version: 1.0
+# uncomment for older versions than stable or nightly
+#robots: noindex
+---
+
+# 1. {{ page.title }}
+
+This plugin enables Foreman to show a nice statistics dashboard and create Trend charts.
```

Comment:
Should we add a banner about this being a wip and not functional yet?

Reply:
I thought we want to do this only after the plugin is done, but I guess it would do no harm to have documentation before code for a change :+1: let me add it :)

Reply:
I've added a callout banner :)